templates/AskdarcelApiService.json

```diff
@@ -90,7 +90,7 @@
 	  "Cpu": "1",
 	  "Essential": "true",
 	  "Image": { "Ref": "ContainerImage" },
-	  "Memory": "300",
+	  "Memory": "1000",
```

Comment:
I did a quick search online, and it looks like there's a way to configure ECS so that the memory limit is just the amount of memory available on the entire instance. This [post](https://forums.aws.amazon.com/thread.jspa?threadID=205442) says that you can use the `memoryReservation` parameter in lieu of the `memory` parameter in order to set a soft limit. According to the [docs](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html), it looks like you only need to set at least one of `memory` or `memoryReservation`, and without a `memory` limit, it'll use up to the amount of memory available on the instance.

Not blocking this PR, but something to look into, especially since we're running single-container instances and not a cluster of containers.